actions/command.js

```diff
@@ -16,7 +16,7 @@ import * as PostActions from 'actions/post_actions.jsx';
 
 import {isUrlSafe, getSiteURL} from 'utils/url.jsx';
 import {localizeMessage, getUserIdFromChannelName} from 'utils/utils.jsx';
-import * as UserAgent from 'utils/user_agent.jsx';
+import * as UserAgent from 'utils/user_agent.tsx';
```

Comment:
We prefer to use imports without the extension for typescript, do you mind to change it everywhere by `utils/user_agent`?

Reply:
Will make changes, thanks!